### PR TITLE
Merge request for the design of the Weather App

### PR DESCRIPTION
**Pull request by sldimitrov**
01. Task from my JS preparation course - cut Figma's design into pieces and code it using HTML and CSS only.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather App</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" type="image/x-icon" href="images/tab-logo.png">
+</head>
+<body>
+
```
Comment (marianserafimov) [inline index.html:11]: Remove all unnecessary white spaces and newline characters

File: style.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    opacity: 0.7;
+}
+
+#searcher {
+    position: relative;
+    top: 15px;
+    left: 200px;
+    width: 25px;
+    height: 25px;
+
```
Comment (marianserafimov) [inline style.css:126]: Remove all unnecessary white spaces and newline characters

File: flexbox/index.html
Comment (marianserafimov) [inline flexbox/index.html:1]: If we are not going to use this file please delete it.

File: flexbox/style.css
Comment (marianserafimov) [inline flexbox/style.css:1]: If we are not going to use this file please delete it.

File: design.png
Comment (marianserafimov) [inline design.png:1]: If we are not going to use this file please delete it.

**Comment by sldimitrov** [comment]
First successfull merge of developed branch within my project

**Pull request merged**